web-app/functions/tickers/requirements.txt

```diff
@@ -0,0 +1,2 @@
+json
+pandas
```

Comment:
need to add `google-cloud` to this file and the `get_reddit_submissions\requirements.txt`